src/content/docs/accounts/original-accounts-billing/original-users-roles/user-migration.mdx

```diff
@@ -48,11 +52,11 @@ For more about these tools, see the [new user management docs](/docs/accounts/ac
 * Once an account’s Owner does the initial step in the wizard of migrating admins, those admins are then able to access and complete the remainder of the migration wizard. 
-* The migration wizard gives you an option to download a full list of your existing original user model users (aka v1 users) before choosing to migrate users. This is recommended because that file can serve as a useful reference tool and, in case it’s needed, as a backup file. 
+* The migration wizard gives you an option to download a full list of your existing original user model users (aka v1 users) before choosing to migrate users. We recommend downloading that file because it can serve as a useful reference and, in case it’s needed, as a backup file. 
 * To check on the status of your user migration, you can go to the [**User management** UI](/docs/accounts/accounts-billing/new-relic-one-user-management/add-manage-users-groups-roles/#where) to see which users are on the new model. 
-* After migration, some users may end up with access to logins for both an original model account and a new model account, both associated with a single email address. This is because our original user model allowed a user, with a single login, to access data from multiple organizations (a common scenario for contractors, for example). But the new user model gives more user control to organizations, and this means that such a user would now require a different login for each organization. If a user previously had access that spanned multiple organizations, they’ll retain an original user record until migration is completed for all associated organizations. One indicator of this situation is that, on login, the UI asks them to [verify their email](/docs/accounts/accounts-billing/account-setup/troubleshoot-new-relics-password-email-address-login-problems/#verify-email) to see all associated accounts, and shows them an “original” account (see below image). 
+* After migration, some users may end up with access to logins for both an original model account and a new model account, both associated with a single email address. This is because our original user model allowed a user to access data from multiple organizations with a single login (a common scenario for contractors, for example). But the new user model gives more user control to organizations, and this means that such a user would now require a different login for each organization. If a user previously had access that spanned multiple organizations, they’ll retain an original user record until migration is completed for all associated organizations. One indicator of this situation is that, on login, the UI asks them to [verify their email](/docs/accounts/accounts-billing/account-setup/troubleshoot-new-relics-password-email-address-login-problems/#verify-email) to see all associated accounts, and shows them an “original” account (see below image). 
```

Comment:
third line rather than "this means that such a user would now require a different login" maybe change to "organizations. For example, these users now need a different login for each organization."
also maybe make a new bullet for "If a user previously had access" etc.
also maybe reword "One indicator of this situation is that, on login, the UI asks them" to "For example, on login, the UI prompts them"